domains/b0ldyyv5.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "B0ldyyV2",
+           "discord": "1332448877965479948",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.GwULSQ3QYawjVuSy5T68eeecdq0-bZobBG5eYolClnX6eK54HGaN8pKpkfzihqBKkjY3bsWdPQI7me0dxUnV-mKUE7EFrjwWx13xcxrqlrU001h-ZY2mvaRt8JIH-pIcS3sL4C9gLjUy5sSHxzJ7dVicR1luElH6HHma6bI_J0Z19JH_nbTeuQHX2HNAsjkoiUwLx20w1EDGho48XhBawMHP0cWQ8hFdH0UOwDyVpmYTG9acQS632qy12G-2478T0QT1hP3UCmCwoKyFhil4vKw3cXlnPmLpGqT0ihJvzWYT0VSt2-zUbeRRfWsq3Sz_EqaSmdd5gHiz7UpXMpDfSw.AafA1a-n64UuUw5hyICYcQ.0rh-GusJV7S5RgH6r-tGcHMZkkrjojOhN13mAe98lpAimGasIVd_5P1srTyEPenFYOfSUDoLvKwueF0-kRookVdxpwF-mkzrO2_l5sCgZes.6x4C9bR0FqjejPkshKlBAw"
+        },
+    
+        "record": {
+            "CNAME": "https://railway.com/u/b0ldyyv2"
```

`CNAME` record must contain a domain name only.
```suggestion
            "CNAME": "railway.com"
```
if you want to make your subdomain redirect to your railway profile, you need to change record type to `URL`
```suggestion
            "URL": "https://railway.com/u/b0ldyyv2"
```
also please read [the docs](https://docs.is-a.dev/).